.travis.yml

```diff
@@ -16,11 +15,6 @@ env:
 matrix:
     allow_failures:
         - php: 7.0
-    include:
-        - php: 5.3
-          env: TEST_SUITE=1 COMPOSER_FLAGS="--prefer-lowest"
-        - php: 5.3
```

Comment:
It may be good idea, but test are passing anyway. I think that we need another PR to discuss version bump related issues. This operation shouldn't be hidden in ThemeBundle :)